### PR TITLE
Create manage-user-sharing-expiration

### PR DESCRIPTION
**Pull request by salarson**
#### Category
- [ ] Content fix
- [ X] New article

#### What's in this Pull Request?

> Please describe the changes in this PR. Sample description or details around bugs which are being fixed.
> Creating a new file for per user sharing expiration which can be handled by CSOM 

#### Guidance

> *Please update this PR information accordingly. We'll use this as part of our release notes in monthly communications.*
> 
> *Please target your PR to 'master' branch. Released documents are in `live` branch.*
>

### CODE REVIEW

File: docs/solution-guidance/manage-user-sharing-expiration [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: Manage per user expiration for existing users on a site
+description: Explains how using client side object model to change a users expiration date for sharing links
+ms.date: 04/01/2020
+ms.prod: sharepoint
+localization_priority: Normal
+---
+
+# Manage per user expiration for existing users on a site
+
+If you are a site administrator, you may need to [manage expiring access](https://https://support.office.com/en-us/article/manage-guest-expiration-for-a-site) for users that have access to content on your site. If your administrator has set an expiration time for access, each guest that you invite to the site or with whom you share individual files and folders will be given access for a certain number of days. If you want them to continue to have access, you must extend their access on a regular basis. There also may be times you want to change an exisiting users expiration time and that can be done via the below method.
```
Comment (andrewconnell) [inline docs/solution-guidance/manage-user-sharing-expiration:11]: Remove locale from URL... let office.com reroute for user's locale.

File: docs/solution-guidance/manage-user-sharing-expiration [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+localization_priority: Normal
+---
+
+# Manage per user expiration for existing users on a site
+
+If you are a site administrator, you may need to [manage expiring access](https://https://support.office.com/en-us/article/manage-guest-expiration-for-a-site) for users that have access to content on your site. If your administrator has set an expiration time for access, each guest that you invite to the site or with whom you share individual files and folders will be given access for a certain number of days. If you want them to continue to have access, you must extend their access on a regular basis. There also may be times you want to change an exisiting users expiration time and that can be done via the below method. 
+
+## Change expiration time for existing user
+
+### Change expiration time for existing user
```
Comment (andrewconnell) [inline docs/solution-guidance/manage-user-sharing-expiration:15]: Dupe headings.. there's no need for the H3... please remove

File: docs/solution-guidance/manage-user-sharing-expiration [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+You can change expiration time for existing user using client side object model:
+
+```powershell
+
```
Comment (andrewconnell) [inline docs/solution-guidance/manage-user-sharing-expiration:20]: Remove extra linebreak

**Comment by andrewconnell** [comment]
@salarson Changes requested were not addressed in the additional commit. Please address the requested changes in the PR before we can approve the doc.

**Comment by andrewconnell** [comment]
Thanks... content looks good now. Because it's new, need to let MSFT merge.

**Comment by andrewconnell** [comment]
@VesaJuvonen LGTM

**Comment by VesaJuvonen** [comment]
Merging and adding to TOC - sorry for the delay here.

**Pull request merged**